src/Agent/NewRelic/Profiler/Configuration/Configuration.h

```diff
@@ -509,7 +509,7 @@ namespace NewRelic { namespace Profiler { namespace Configuration {
         {
             auto isIis = Strings::EndsWith(processName, _X("W3WP.EXE")) || Strings::EndsWith(parentProcessName, _X("W3WP.EXE"));
 
-            LogInfo(_X("Process ") + processName + _X(" with parent process ") + parentProcessName + (isIis ? _X(" is") : _X(" is not")) + _X(" IIS."));
+            LogInfo(_X("Process ") + processName + (parentProcessName == _X("") ? _X("") : _X(" with parent process ") + parentProcessName) + (isIis ? _X(" is") : _X(" is not")) + _X(" IIS."));
```

Comment:
Would it be clearer if we said something like `"with unknown parent process"` for this case? Or would that be more confusing?

Reply:
I feel like that could still lead people to conclude that we would like to know the parent process in this case but for some reason can't get that info.